kong/plugins/request-transformer/access.lua

```diff
@@ -1,5 +1,5 @@
 local multipart = require "multipart"
-local cjson = require "cjson"
+local cjson = require("cjson.safe").new()
```

Comment:
If you're going to switch from `cjson` to `cjson.safe` then you need to update the `parse_json` local function below, as it uses `pcall`.